webapp/certified/views.py

```diff
@@ -0,0 +1,352 @@
+import talisker.requests
+import talisker.sentry
+import requests
+import math
+from flask import request, render_template, abort, current_app
+from requests import Session
+from webapp.certified.api import CertificationAPI
+from webapp.certified.helpers import get_download_url
+
+session = Session()
+talisker.requests.configure(session)
+api = CertificationAPI(
+    base_url="https://certification.canonical.com/api/v1", session=session
+)
+
+
+def certified_component_details(component_id):
+
+    try:
+        component = api.component_summary(component_id)
+    except requests.exceptions.HTTPError as error:
+        if error.response.status_code == 404:
+            abort(404)
+        else:
+            current_app.extensions["sentry"].captureException()
```

Comment:
I notice here the exception is captured for sentry but not in other places. is that intentional?

Reply:
I think in this case we can only do this if there is an HTTP error. But Sentry will catch any 500 errors anyway.